src/Auth/HumanInteractionInterface.js

```diff
@@ -0,0 +1,30 @@
+class HumanInteractionInterface {
+
+  constructor(auth) {
+    this.auth = auth; 
+  }
+  
+  /**
+   * Called at the end of AuthController.init()
+   * - Load potential assets 
+   * - Register state change with
+   * this.auth.store.stateChangeListners.push(this.onStateChange.bind(this));
+   * implementation is optional
+   */ 
+  async init () { }
+
+  /**
+   * Called each time the Auth state change
+   */
+  onStateChange () {
+    throw new Error('onStateChange() must be implemented')
+  }
+
+  /**
+   * Not mandatory, but needed to launch auth request and
+   * start authentication popup
+   */
+  onClick () {}
```

Comment:
do we leave this in the interface? if yes, we need to adapt the readme example https://github.com/pryv/lib-js/tree/refactor/modular-auth#using-a-custom-login-button

Reply:
Don't know. As we discussed it is here only as an example but not as a required function to implement.

Reply:
fine.